portfolio/src/main/webapp/style.css

```
@@ -7,3 +7,108 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.center {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
+}
+body {
+    background-image: url('https://img.freepik.com/free-photo/white-concrete-wall_53876-92803.jpg');
+}
+h1 {text-align: center;}
+*{
+padding:0;
+margin:0;
+text-decoration: none;
+list-style: none;
+box-sizing: border-box;
+}
+
+body{
```

[4.2.5](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation): Use a space between the last selector and the declaration block.

```suggestion
body {
```

Here and elsewhere